.github/workflows/integrations.yml

```diff
@@ -382,3 +382,49 @@ jobs:
         with:
           slack-webhook: ${{ secrets.E2E_INTEGRATIONS_SLACK_WEBHOOK }}
           job-status: ${{ needs.detectron2.result }}
+
+  sklearn:
+    env:
+      NEPTUNE_PROJECT: e2e-tests/integrations
+    timeout-minutes: 75
+    strategy:
+      fail-fast: false
+      matrix:
+        python-version: [ "3.7", "3.9" ]
+        os: [ ubuntu-latest, macos-latest ]
```

Comment:
How about `windows`? I saw that in https://github.com/neptune-ai/neptune-sklearn we're running tests on all operating systems.

Reply:
Yes. Not sure how I skipped it. Fixed.